editor/plugins/mesh_instance_3d_editor_plugin.cpp

```diff
@@ -630,12 +855,39 @@ MeshInstance3DEditor::MeshInstance3DEditor() {
-	shape_type->set_item_tooltip(-1, TTR("Creates a polygon-based collision shape.\nThis is a performance middle-ground between a single convex collision and a polygon-based collision."));
+	shape_type->set_item_tooltip(-1, TTR("Creates multiple convex collision shapes. These are decomposed from the original mesh.\nThis is a performance and accuracy middle-ground between a single convex collision and a polygon-based trimesh collision."));
+	shape_type->add_item(TTR("Bounding Box"), SHAPE_TYPE_BOUNDING_BOX);
+	shape_type->set_item_tooltip(-1, TTR("Creates an bounding box collision shape.\nThis will use the mesh's AABB if the shape is not a built-in BoxMesh.\nThis is faster than the convex collision shape option for collision detection."));
+	shape_type->add_item(TTR("Capsule"), SHAPE_TYPE_CAPSULE);
+	shape_type->set_item_tooltip(-1, TTR("Creates a capsule collision shape.\nThis will use the mesh's AABB if the shape is not a built-in CapsuleMesh.\nThis is faster than the convex collision shape option for collision detection."));
+	shape_type->add_item(TTR("Cylinder"), SHAPE_TYPE_CYLINDER);
+	shape_type->set_item_tooltip(-1, TTR("Creates a cylinder collision shape.\nThis will use the mesh's AABB if the shape is not a built-in CylinderMesh.\nThis is faster than the convex collision shape option for collision detection."));
+	shape_type->add_item(TTR("Sphere"), SHAPE_TYPE_SPHERE);
+	shape_type->set_item_tooltip(-1, TTR("Creates a sphere collision shape.\nThis will use the mesh's AABB if the shape is not a built-in SphereMesh.\nThis is faster than the convex collision shape option for collision detection."));
+	shape_type->add_item(TTR("Primitive"), SHAPE_TYPE_PRIMITIVE);
+	shape_type->set_item_tooltip(-1, TTR("Creates a box, capsule, cylinder, or sphere primitive collision shape if the mesh is a primitive.\nThe mesh must use the built-in BoxMesh, CapsuleMesh, CylinderMesh, or SphereMesh primitive.\nThis is faster than the convex collision shape option for collision detection."));
 	shape_dialog_vbc->add_child(shape_type);
+	shape_type->connect(SceneStringName(item_selected), callable_mp(this, &MeshInstance3DEditor::_shape_type_selected));
+
+	shape_axis_label = memnew(Label);
+	shape_axis_label->set_text(TTR("Alignment Axis"));
+	shape_dialog_vbc->add_child(shape_axis_label);
+
+	shape_axis = memnew(OptionButton);
+	shape_axis->set_h_size_flags(SIZE_EXPAND_FILL);
+	shape_axis->add_item(TTR("Longest axis"), SHAPE_AXIS_LONGEST);
+	shape_axis->set_item_tooltip(-1, TTR("Create the shape along the longest axis of the mesh's AABB."));
+	shape_axis->add_item(TTR("X axis"), SHAPE_AXIS_X);
+	shape_axis->set_item_tooltip(-1, TTR("Create the shape along the local X axis."));
+	shape_axis->add_item(TTR("Y axis"), SHAPE_AXIS_Y);
+	shape_axis->set_item_tooltip(-1, TTR("Create the shape along the local Y axis."));
+	shape_axis->add_item(TTR("Z axis"), SHAPE_AXIS_Z);
+	shape_axis->set_item_tooltip(-1, TTR("Create the shape along the local Z axis."));
```

Comment:
Use Title Case for option names for consistency with the rest of the editor:

```suggestion
	shape_axis->add_item(TTR("Longest Axis"), SHAPE_AXIS_LONGEST);
	shape_axis->set_item_tooltip(-1, TTR("Create the shape along the longest axis of the mesh's AABB."));
	shape_axis->add_item(TTR("X Axis"), SHAPE_AXIS_X);
	shape_axis->set_item_tooltip(-1, TTR("Create the shape along the local X axis."));
	shape_axis->add_item(TTR("Y Axis"), SHAPE_AXIS_Y);
	shape_axis->set_item_tooltip(-1, TTR("Create the shape along the local Y axis."));
	shape_axis->add_item(TTR("Z Axis"), SHAPE_AXIS_Z);
	shape_axis->set_item_tooltip(-1, TTR("Create the shape along the local Z axis."));
```

Reply:
Looking through some code, I think `X-Axis` is more preferred.